tests/unit_test/channels/bsp_test.py

```
@@ -1297,3 +1297,82 @@ async def test_upload_media_file_non_200_response(self, tmp_path):
+
+    mock_http.assert_called_once()
+    assert result == "Media file deleted successfully"
+
+
+def test_delete_media_file_not_exist_raises():
+    media_id = "12345"
+    channel_config = {"config": {"api_key": "dummy_api_key"}}
+
+    with patch("kairon.shared.utils.Utility.execute_http_request") as mock_http:
+        mock_http.side_effect = AppException("media file does not exist for this media id.")
+        with pytest.raises(AppException, match="media file does not exist for this media id."):
```

Ruff (0.13.1)

Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)